publications.Rmd

```diff
@@ -48,6 +51,22 @@ for (yeari in as.character(2022:2019)) {
 
 ## Conference Presentations
 
+### 2022
+
+Lai, M. H. C., Tse, W. W. Y., Zhang, G., Li, Y., & Hsiao, Y. Y. (2022, September 21-24). *Correcting for unreliability and partial invariance: A two-stage path analysis approach* [Symposium presentation]. Society for Research on Educational Effectiveness 2022 Conference, Arlington, VA, United States. https://www.psychometricsociety.org/sites/main/files/file-attachments/imps2022-version-7-abstract.pdf
+
+Zhang, G., & Lai, M. H. C. (2022, July 11-15). *A systematic review of measurement invariance research of the CES-D across gender: calculation and report of effect size* [Oral presentation]. Annual Meeting of the Psychometric Society (IMPS), Bologna, Italy. https://www.psychometricsociety.org/sites/main/files/file-attachments/imps2022-version-7-abstract.pdf
+
+Tse, W. W., & Lai, M. H. C. (2022, July 11-15). *Incorporating information from historical data for power analysis: A hybrid classical-Bayesian approach for multilevel studies *[Oral presentation]. Annual Meeting of the Psychometric Society (IMPS), Bologna, Italy. https://www.psychometricsociety.org/sites/main/files/file-attachments/imps2022-version-7-abstract.pdf
+
+Zhang, Y., & Lai, M. H. C. (2022, July 11-15). *Bayesian region of measurement equivalence (ROME) approach with alignment *[Oral presentation]. Annual Meeting of the Psychometric Society (IMPS), Bologna, Italy. https://www.psychometricsociety.org/sites/main/files/file-attachments/imps2022-version-7-abstract.pdf
+
```

Comment:
I realize I forgot to add this conference presentation: 
Zhang, Y., & Lai, M. H. C. (2022, July 11-15). *Evaluating standard error estimators and resampling method for multilevel models on small samples with heteroscedasticity and unbalanced cluster sizes *[Poster presentation]. Annual Meeting of the Psychometric Society (IMPS), Bologna, Italy.